test/unit/deadline_mcp/test_server.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""Unit tests for the AWS Deadline Cloud MCP server."""
+
+import json
+import os
+import pytest
+from datetime import datetime
+from unittest.mock import patch, MagicMock
+
+from deadline.mcp.tools.job import submit_job
+
+
+class TestMCPServer:
```

Comment:
It'd be great to have a test which starts the real MCP server, then 1) uses an MCP client to connect to call ListTools and 2) actually invokes a tool and gets back a valid response.